requirements.prod.in

```diff
@@ -14,6 +14,7 @@ incuna-mail
 itsdangerous
 litecli
```

Comment:
It won't be once the changes are complete, but leaving in until we're definitely moved over in case we need to jump into the SQLite db to check something.